apps/token/src/routes/rewards/epoch-individual-rewards/epoch-individual-rewards-table.tsx

```diff
@@ -0,0 +1,102 @@
+import { formatNumber, toBigNum } from '@vegaprotocol/react-helpers';
+import { Tooltip } from '@vegaprotocol/ui-toolkit';
+import { useAppState } from '../../../contexts/app-state/app-state-context';
+import {
+  rowGridItemStyles,
+  RewardsTable,
+} from '../shared-rewards-table-assets/shared-rewards-table-assets';
+import type { EpochIndividualReward } from './generate-epoch-individual-rewards-list';
+import { useTranslation } from 'react-i18next';
+
+interface EpochIndividualRewardsGridProps {
+  data: EpochIndividualReward;
+}
+
+interface RewardItemProps {
+  value: string;
+  percentageOfTotal?: string;
+  dataTestId: string;
+  last?: boolean;
+}
+
+const DisplayReward = (reward: string, percentageOfTotal?: string) => {
+  const { t } = useTranslation();
+  const {
+    appState: { decimals },
+  } = useAppState();
```

Comment:
Suggest making this a prop so this component is stupider

Reply:
It means passing props around more often than we need as the only component that needs decimals is this one. If this is just a suggestion, I think I'll leave it, but let me know if you feel it's a problem.